docs/articles/documentation/using-testcafe/common-concepts/browser-support.md

```diff
@@ -29,33 +33,38 @@ Mozilla Firefox   | 17+    | `firefox`
+You can specify local browsers by using *paths* or *browser aliases* (see the table above).
+The list of all the available browsers can be obtained by calling the [--list-browsers](../command-line-interface.md#-b---list-browsers) command.
 
-If you need to use a browser that is not [supported directly](#directly-supported-browsers), create a [browser provider plugin](../../extending-testcafe/browser-provider-plugin/).
+## Portable Browsers
 
-Created provider packages can be published to npm. The provider package name consists of two parts - the `testcafe-browser-provider-` prefix and the name of a provider itself; e.g., `testcafe-browser-provider-saucelabs`.
-So, you can search for available browser provider packages on npm by the `testcafe-browser-provider-` prefix: [https://www.npmjs.com/search?q=testcafe-browser-provider](https://www.npmjs.com/search?q=testcafe-browser-provider).
+Supported portable browsers:
 
-For example, there are provider packages for testing in [Sauce Labs](https://saucelabs.com/) and [PhantomJS](http://phantomjs.org/) browsers:
+* Google Chrome Portable
+* Firefox Portable
 
-* [testcafe-browser-provider-saucelabs](https://www.npmjs.com/package/testcafe-browser-provider-saucelabs)
-* [testcafe-browser-provider-phantomjs](https://www.npmjs.com/package/testcafe-browser-provider-phantomjs)
+You can specify them by using *paths* to browser executable files.
 
-You can install browser provider packages from npm as you would install any other plugin. See [Installing Plugins](../../extending-testcafe/index.md#installing-plugins).
+## Remote Browsers
 
-## Specifying Browsers for Test Task
+TestCafe can run tests in browsers on remote devices. The only requirement is a device should have network access to the TestCafe server.
 
-If you are working with TestCafe through the command line,
-[list the desired browsers](../command-line-interface.md#browser-list)
-within the `testcafe` command.
+At first, you need to create a remote browser connection. You can do this from CLI by specifying the `remote` *alias* or from API by using the [createBrowserConnection](../programming-interface/testcafe.html#createbrowserconnection) method.
 
-In case you are using the API, specify the browsers
-with the [runner.browsers](../programming-interface/runner.md#browsers) method.
+After that, TestCafe will provide URLs that you need to open in required browsers on your remote device to start testing.
+
+## Cloud Browsers
+
+TestCafe allows you to use cloud browsers that can be accessed through *browser provider plugins*.
 
-To identify browsers, different approaches are used.
+For example, there are browser provider plugins for testing in [Sauce Labs](https://saucelabs.com/) and [PhantomJS](http://phantomjs.org/) browsers:
+
+* [testcafe-browser-provider-saucelabs](https://www.npmjs.com/package/testcafe-browser-provider-saucelabs)
+* [testcafe-browser-provider-phantomjs](https://www.npmjs.com/package/testcafe-browser-provider-phantomjs)
```

Comment:
PhantomJs is not a cloud browser. You can run it locally on your machine with this plugin